### DISCUSSION TITLE
Java error

### DISCUSSION THREAD

**Discussion by Esteban528** [LazyVim/LazyVim#4134] [Q&A]
Hello, I have this error when jumping line in java, I get this error, I don't know if it is some nvim-java, lsp, or lazyVim problem, I appreciate any help

I'm using this plugin: https://github.com/nvim-java/nvim-java

### Error:
![image](https://github.com/user-attachments/assets/556b769e-4ea2-4689-a729-f7ac29f2c3cb)

### My Config
![image](https://github.com/user-attachments/assets/00e9962a-d0f6-46e8-8e62-e992abf4e916)

**Comment by iamxiaojianzheng** [comment]
Try turning off inlay hints using `<leader>uh`.

Do not close lazyvim, reopen the java file directly through ':e'.

**Comment by Esteban528** [comment]
I turned off the inlay hints and it worked fine, thank you very much

**Reply by rogeriofrsouza** [reply]
Could you report this issue in `nvim-java` repository?
Seems like a problem with the plugin compatibility in Neovim v0.10. The maintainer could fix this.

**Reply by Esteban528** [reply]
I did, I hope the maintainer can read it
